### PR TITLE
[now dev] Improve integration tests for `now dev`

### PR DESCRIPTION
**Pull request by thasophearak**
This PR using `stdio` to resolve `now dev` ready state.

### CODE REVIEW

File: test/dev/integration.js [hunk 1]
```diff
@@ -54,6 +54,25 @@ function testFixture(directory) {
...
+      port = ++port;
+      const dev = execa(binaryPath, ['dev', directory, '-p', port]);
+
+      dev.stderr.on('data', async data => {
+        if (data.toString().includes('Ready! Available at')) {
+          resolve({dev, port})
+        }
+      });
+
+      await dev;
```
Comment (AndyBitz) [inline test/dev/integration.js]: Why is this needed?

File: test/dev/integration.js [hunk 2]
```diff
@@ -54,6 +54,25 @@ function testFixture(directory) {
...
 
+function testFixtureStdio(directory) {
+  return new Promise(async (resolve, reject) => {
+    try {
+      port = ++port;
+      const dev = execa(binaryPath, ['dev', directory, '-p', port]);
+
+      dev.stderr.on('data', async data => {
+        if (data.toString().includes('Ready! Available at')) {
+          resolve({dev, port})
```
Comment (AndyBitz) [inline test/dev/integration.js]:
```suggestion
          resolve({ dev, port })
```

File: test/dev/integration.js (hunk 1 above)
Comment (thasophearak) [reply test/dev/integration.js]: @AndyBitz Thank for spot it, indeed, it isn't necessary here. removed it.

**Comment by TooTallNate** [comment]
The tests are passing. What's the status on this one @sophearak?

**Comment by thasophearak** [comment]
@TooTallNate There are few more tests I want to include from example that recently added https://github.com/zeit/now-examples

Should be able to wrap this up today.

**Review by TooTallNate — APPROVED** [review]
